docs/installation_windows.md

```diff
@@ -0,0 +1,87 @@
+# Install LISA on Windows
+
+- [Install Python on Windows](#install-python-on-windows)
+- [Install dependencies on Windows](#install-dependencies-on-windows)
+- [Clone code](#clone-code)
+- [Install Poetry on Windows](#install-poetry-on-windows)
+- [Verify installation](#verify-installation)
+- [FAQ and Troubleshooting](#faq-and-troubleshooting)
+
+:blue_book: On Windows, after you finished an installation, or made an
+environment variable change, you might need to restart your shell before moving
+to next step, to make sure your changes take effect.
+
+:blue_book: Please run your command prompt or shell with elevated privilege
+(such as `'Run as Administrator'` on Windows) when you see access denied message
+when install tools.
+
+## Install Python on Windows
+
+LISA has been tested on [Python 3.8 64 bits](https://www.python.org/). The
+latest version of Python 3 is recommended. If you found LISA is not compatible
+with higher version Python, [please file an
+issue](https://github.com/microsoft/lisa/issues/new).
+
+Navigate to [Python releases for
+Windows](https://www.python.org/downloads/windows/). Download and install
+*Windows installer (64-bit)* for Python 3.8 64-bits or higher version.
+
+:warning: Please make sure the `Python` directory and its `Scripts` directory
+are added to your `PATH` environment variable. For example:
+
+```powershell
+PS C:\github> echo $env:path
```

Comment:
It should specify it's the output example, not command example. I'm not sure if some users doesn't check it correctly, they will use it like commands.